src/translate/instantiate.py

```diff
@@ -53,7 +53,7 @@ def instantiate_goal(goal, init_facts, fluent_facts):
 
 # The input task must have been normalized
 # The model has been computed by build_model.compute_model
-def instantiate(task: pddl.Task, model: Any) -> Tuple[
+def instantiate(task: pddl.Task, model: Any, filter_noop_operators: bool) -> Tuple[
```

Comment:
At the moment it seems that the prevailing way we use the `options` module is by accessing the options directly where needed, as in importing `options` and using `options.filter_noop_operators` like a global variable. It's a questionable design choice regarding encapsulation, and there are one or two places where we handle things a bit differently, but I think at this point it's better to keep the code consistent in that style. So I would get rid of changes like this one that are about passing the option through. For example, in this file, this would mean undoing all changes.